### ISSUE TITLE
Japanese File Name Support?

### ISSUE THREAD

**Issue by JanetMerai** [libretro/RetroArch#5448]
I'm running Windows 10, 64 bit edition and using Japanese file names in the actual games I am emulating.
I tested by renaming a file with Kanji and Kana in it to something like 'test.fds' and it read the file.
When I tested a game with English in the file name, it loaded the game fine, but in actual Japanese characters, it didn't.

Will there be support for non Latin / English directory and file name character support?

FDS means Famicom Disk System.

**Comment by gouchi** [comment]
Which RA version are using ? 

Can you provide a screenshot of the issue.

Thank you.

**Comment by bparker06** [comment]
duplicate of #5019

**Comment by JanetMerai** [comment]
I am using 1.6.7, and when I load a file with Japanese characters in it, it tells me it cannot load the file.

![screen shot 09-20-17 at 01 58 pm](https://user-images.githubusercontent.com/4703831/30664792-206f8836-9e0c-11e7-85be-e0e489aa2efd.PNG)
![screen shot 09-20-17 at 02 00 pm](https://user-images.githubusercontent.com/4703831/30664797-259e680e-9e0c-11e7-8cf7-7b9fb883e220.PNG)

That's what I get.

**Comment by bparker06** [comment]
I am about to submit a PR to fix this, but closing this issue now since it is a duplicate anyway.

**Comment by JanetMerai** [comment]
This update works great! :D

The only problem I found, compared to the Android port, is loading Japanese file-paths appear to be not working.
In the case of Android, it worked with Japanese file-paths, but on Windows 10, it doesn't.

I tested loading a path with English only and it works, but folder paths with Japanese characters and Kanji in it makes it fail to load the content.

Am I doing something wrong?
